src/components/map-container.js

```diff
@@ -439,23 +462,25 @@ export default function MapContainerFactory(MapPopover, MapControl) {
       return (
         <StyledMapContainer style={MAP_STYLE.container}>
           <MapControl
+            index={index}
```

Comment:
we already pass in`mapIndex={this.props.index}`, no need to pass in`index={index}`

Reply:
yeah, I already removed that one